themes/survey/vanilla/scripts/ajaxify.js

```diff
@@ -111,7 +111,7 @@ var AjaxSubmitObject = function () {
         // Restrict to [type=submit]:not([data-confirmedby])
         // - :submit is the default if button don't have type (reset button on slider for example),
         // - confirmedby have their own javascript system
-        $(document).on('click', '#ls-button-submit, #ls-button-previous', function (e) {
+        $(document).on('click', '#limesurvey [type=submit]:not([data-confirmedby]:not(disabled))', function (e) {
```

Comment:
Why you are so keen on having this selector? I still strongly oppose this.
It is not specific enough and the effects of pagetriggers should always only be done by page triggers.
One way around it is to add a specific selector class just for this kind of purpose? Would you be ok with that?
It is a simple change, just use i.g. .selector__trigger-page-submit to select the triggering buttons.

Reply:
A data , not a class. data-action='submit' for example. But i really think the purpose (action='submit') of a button must launch submit action … this action was done without js, then same action must be done with js …

Currently : something working on 2.73 + answer_html + 2.6lts didn't work in master. 

And the current system for disable multiple submit didn't work : https://bugs.limesurvey.org/view.php?id=13911

Reply:
Yes, and … deactivate ajaxmode : click on Index button : it work. Activate ajaxmode, click on index button : it don't work …